Automattic/sensei
# Add tasks data

Part of #5633

### Changes proposed in this Pull Request

* It adds the tasks' actual data to the Sensei Home.
* It renames the `tasks` folder to `tasks-section`.

In order to make the PRs smaller, some features will be included in other PRs:

* It still doesn't get the course data (side illustration).
  * The ready state still doesn't have the correct social links.
* It still doesn't have the completion logic (the animation and call to the complete endpoint when all tasks are completed).

### Testing instructions


* In a new env, complete task by task, go to the Sensei Home, and make sure it reflects the correct status
  * Notice that it's not marking all the tasks as completed yet, but if you force it, you will see the Ready state.
